tests/listobjects/listobjects_test.go

```diff
@@ -27,6 +27,8 @@ func testRunAll(t *testing.T, engine string) {
 	cfg := run.MustDefaultConfigWithRandomPorts()
 	cfg.Log.Level = "none"
 	cfg.Datastore.Engine = engine
+	cfg.AllowWriting1_0Models = true
+	cfg.AllowEvaluating1_0Models = true
```

Comment:
i'm not particularly happy that we are setting this to true for every single test, even though it should only be needed for the 1.0 tests, but i'll let it slide 😆

Reply:
:)